server/routes/ChallengeRoutes.js

```diff
@@ -1,6 +1,8 @@
 const Challenge = require("../controllers/ChallengeController");
 
 const router = require("express").Router();
+
+const checkRole = require('./middleware/checkRole');
```

Comment:
where is the middleware directory?
Not showing up in the commit

Reply:
I made it in the later commits